src/Views/batchprocessor.html

```diff
@@ -104,16 +136,53 @@ <h3>Select Basin Characteristics</h3>
                         </tr>
                     </tbody>
                 </table>
+            </div><!-- End parameter selection -->
 
-            </div>
-            <!-- End submit batch -->
+            <!-- Start user information -->
+            <div style="padding-top: 15px;" class="row">
+                <div class="col-md-5"> ID Field: 
+                    <i ng-mouseover="showTooltip = true" ng-mouseleave="showTooltip = false" class="fa fa-question-circle"></i>
+                    <span ng-show="showTooltip" class="report-tooltip">
+                        <span>This is the field heading from the shapefile of points that is unique to each point and allows the

```

Comment:
When this text appears, it pushes down the input box, which looks a little funny. Is it possible to have this appear in a floating text box, like seen here ([image](https://user-images.githubusercontent.com/40237491/230174870-a26acf7d-8c08-49d9-893a-d00ce4d7af2e.png))?

Reply:
I made a comment about this in #1460:
Note: tooltip functionality through class="report-tooltip" should be the same as SCRunoff when both batchprocessor and SCRunoff are merged into SS. Confirmed by adding copying .report-tooltip from SCRunoff branch to working branch in BP in main.css. After confirmation, discarded changes to main.css in batchprocessor branch.

I didn't want to duplicate the additions to main.css from SCRunoff but can and open an issue to remove it when both are part of main.